Library/Homebrew/cmd/update-report.rb

```diff
@@ -50,6 +50,11 @@ def update_report
   def output_update_report
     args = update_report_args.parse
 
+    # Check if we can parse the JSON and do any Ruby-side follow-up.
+    ENV["HOMEBREW_NO_AUTO_UPDATE"] = "1" # We just downloaded so no need to do so again
```

Comment:
This feels a bit weird, won't the `mtime` stop another download attempt anyway? If it doesn't: it probably should.

Reply:
@Bo98 Yeh, probably need to touch there too. It does make me wonder if both sides should just call a shared Bash command so this doesn't keep getting out of sync...

Reply:
Yeah I feel so too but will leave any refactoring for a separate PR.

Reply:
@Bo98 Yeh, cool. Think it's probably worth doing the `touch` here instead of this, though.